gouttelette/data/plugins/module_utils/core.py

```diff
@@ -41,8 +41,6 @@
     scrub_keys,
     to_sync,
     to_async,
-    ansible_dict_to_boto3_tag_list,
-    snake_dict_to_camel_dict,
```

Comment:
We need to keep these because they are used by each generated module.

Reply:
There are linter failures due to unused imports.

Reply:
And these are imports used by the functions in this file. I am not sure how it will impact the generated modules. Please correct me if my understanding is wrong.

Reply:
These imports are used here https://github.com/ansible-collections/amazon_cloud_code_generator/blob/main/amazon_cloud_code_generator/templates/default_module.j2 and then reflected in each module, that's why we need them.